Tools/LyTestTools/tests/unit/test_o3de_editor_test.py

```diff
@@ -648,6 +648,8 @@ def test_ExecEditorTest_TestCrashes_ReturnsCrash(self, mock_cycle_crash, mock_ge
         assert isinstance(results[mock_test_spec.__name__], editor_test.Result.Crash)
         assert mock_editor.start.called
         assert mock_retrieve_crash.called
+        # Save editor log, crash log, and crash dmp
+        assert mock_workspace.artifact_manager.save_artifact.call_count == 3
```

Comment:
Won't this test fail on non-Windows platforms since the crash dmp won't exist and won't get saved?

Reply:
The test is mocked for the file existing so it's platform agnostic